en/releases/1.12.md

```diff
@@ -22,6 +30,27 @@
 * **Airframe Scripts**
   * The syntax for setting defaults was changed and custom scripts require an update
   * See [PR#16796](https://github.com/PX4/PX4-Autopilot/pull/16796/files#diff-dcf2f5536f47f260e5e0ff3b3fd22eaef6b6c510126463d70affa0eb7bd4d3ddL20) for an example.
+* Safety (switch) defaults to off (motors are disarmed, but servos / flaps can move)
+* Safety switch is latching: Once it is disabled, it will stay disabled
+* Added support for IRC Ghost including telemetry
+
+### Sensors
+* Magnetometer calibration faster and more robust
```

Comment:
This is a big deal, and if significantly faster worthy of a blog.

Does the process change at all? Do any of the documented existing parameters change?